protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -318,6 +325,22 @@ func (r *Registry) SetMergeFileName(mergeFileName string) {
 	r.mergeFileName = mergeFileName
 }
 
+// SetAllowRepeatedFieldsInBody controls whether repeated field can be used in body field path
```

Comment:
Given you are providing a getter for `useJSONNameInSwaggerDef`, did you mean to provide it for `allowRepeatedFieldsInBody`?